hydrolib/core/io/fnm/models.py

```diff
@@ -20,8 +21,8 @@ class RainfallRunoffModel(FileModel):
 
     # Note that order is defined by the .fnm file type and is used for parsing the data.
     control_file: Path = Path("delft_3b.ini")
-    node_data: Optional[NodeFile]
-    link_data: Optional[LinkFile]
+    node_data: Optional[NodeFile] = Field(None)
+    link_data: Optional[LinkFile] = Field(None)
```

Comment:
I would make sure to cover the initialization of this class with tests. (goes as well for bui_file).